src/test/kotlin/com/example/delivery/service/OrderServiceTest.kt

```diff
@@ -0,0 +1,96 @@
+package com.example.delivery.service
+
+import com.example.delivery.OrderFixture.createOrderDTO
+import com.example.delivery.OrderFixture.domainOrder
+import com.example.delivery.OrderFixture.order
+import com.example.delivery.ProductFixture.product
+import com.example.delivery.UserFixture.user
+import com.example.delivery.dto.request.UpdateOrderDTO
+import com.example.delivery.exception.NotFoundException
+import com.example.delivery.mongo.MongoOrder
+import com.example.delivery.repository.OrderRepository
+import com.example.delivery.repository.ProductRepository
+import com.example.delivery.repository.UserRepository
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import org.junit.jupiter.api.extension.ExtendWith
+import org.mockito.InjectMocks
+import org.mockito.Mock
+import org.mockito.Mockito
+import org.mockito.junit.jupiter.MockitoExtension
+import org.mockito.kotlin.any
+import org.mockito.kotlin.times
+import org.mockito.kotlin.verify
+import kotlin.test.assertEquals
+
+@ExtendWith(MockitoExtension::class)
+internal class OrderServiceTest {
+    @Mock
+    private lateinit var orderRepository: OrderRepository
+
+    @Mock
+    private lateinit var productRepository: ProductRepository
+
+    @Mock
+    private lateinit var productReserveService: ProductReserveService
+
+    @Mock
+    private lateinit var userRepository: UserRepository
+
+    @InjectMocks
+    private lateinit var orderService: OrderService
+
+    @Test
+    fun `should return order when order exists`() {
+        //GIVEN
+        Mockito.`when`(orderRepository.findById("1")).thenReturn(order)
+
+        //THEN
+        assertEquals(orderService.findById("1"), domainOrder)
+    }
+
+    @Test
+    fun `should throw exception when order doesn't exists while find`() {
+        //GIVEN
+        Mockito.`when`(orderRepository.findById("1")).thenReturn(null)
+
+        //THEN
```

Comment:
```suggestion
        // WHEN //THEN
```

don't forget about the most important `when` comment. This test a bit confusing at first look, because it's not clear what we test here